Makefile

```diff
@@ -9,17 +9,25 @@ PREFIX ?= ${ROOT}/usr
 INSTALL_DIR ?= ${PREFIX}
 MODULE_DIR := ${ROOT}/modules
 
-DEB_COMPONENT := clearwater-snmp-handlers
-DEB_MAJOR_VERSION := 1.0${DEB_VERSION_QUALIFIER}
+PKG_COMPONENT := clearwater-snmp-handlers
+PKG_MAJOR_VERSION ?= 1.0
+
+# Define DEB_NAMES and RPM_NAMES separately as we don't have RPMs for all the
+# components yet.
 DEB_NAMES := clearwater-snmp-handler-cdiv clearwater-snmp-alarm-agent clearwater-snmp-handler-memento-as clearwater-snmp-handler-memento clearwater-snmp-handler-astaire clearwater-snmp-alarm-agent-dbg
+RPM_NAMES := clearwater-snmp-alarm-agent clearwater-snmp-alarm-agent-debuginfo
```

Comment:
Should we changing DEB_NAMES to PKG_NAMES and make cw-rpm.mk be smart about "-dbg" -> "-debuginfo" (or automatically look for "-dbg"/"-debuginfo" versions of every package, as they're always going to be tied to the same repository as their parent)?